### PR TITLE
fix(client): build page navigation to service

### PR DESCRIPTION
**Pull request by sanskar-soni-9**
Close: #7477 

## PR Details

The PR Solves issue #7477 Providing a better navigation on Build page and Commit page according to the design.

## PR Checklist

- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-client/src/VersionControl/BuildPage.scss [hunk 1]
```diff
@@ -4,6 +4,25 @@ $log-min-width: 360px;
 $icon-size: 20px;
 
 .build-page {
+  &__header {
+    gap: 24px;
+    margin-bottom: 16px;
```
Comment (abrl91) [inline packages/amplication-client/src/VersionControl/BuildPage.scss:9]: Please don't use hard-coded values.
For the margin `16px` you can use `var(--large-spacing)` if I'm not mistaken.
For the gap you can set a local var like in line 4.

File: packages/amplication-client/src/VersionControl/CommitResourceListItem.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      color: #ffffff;
+    }
+  }
+
+  &__description-row {
+    @include regular-10;
+
+    .amp-text {
+      font-size: 10px;
+      line-height: 15px;
```
Comment (abrl91) [inline packages/amplication-client/src/VersionControl/CommitResourceListItem.scss:25]: use variables

**Review by abrl91 — CHANGES_REQUESTED** [review]

File: packages/amplication-client/src/VersionControl/CommitResourceListItem.scss (hunk 2 above)
Comment (sanskar-soni-9) [reply packages/amplication-client/src/VersionControl/CommitResourceListItem.scss:25]: ok sure, can i convert it to draft and mix it or create new pr with the fix.

**Comment by sanskar-soni-9** [comment]
@abrl91 Done, let me know if any more changes are required. Thanks :)

File: packages/amplication-client/src/VersionControl/BuildPage.scss (hunk 1 above)
Comment (sanskar-soni-9) [reply packages/amplication-client/src/VersionControl/BuildPage.scss]: Resolved with `var(--large-spacing)` for `16px` and `var(--double-spacing)` for `24px`.

**Comment by abrl91** [comment]
@barshimi @morhag90 can you review, please?

File: packages/amplication-client/src/VersionControl/CommitResourceListItem.tsx [hunk 3]
```diff
@@ -42,51 +45,97 @@ const CommitResourceListItem = ({ build, commitChangesByResource }: Props) => {
...
+          className={`${CLASS_NAME}__description-row`}
+        >
+          <HorizontalRule smallSpacing />
+          <FlexItem
+            itemsAlign={EnumItemsAlign.Center}
+            end={
+              <FlexItem itemsAlign={EnumItemsAlign.Center}>
+                <Link
+                  to={`/${currentWorkspace?.id}/${currentProject?.id}/${build.resourceId}/changes/${build.commitId}`}
+                  target="_blank"
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/VersionControl/CommitResourceListItem.tsx:89]:
```suggestion
```

File: packages/amplication-client/src/VersionControl/BuildPage.tsx [hunk 4]
```diff
@@ -97,48 +110,56 @@ const BuildPage = ({ match, buildId }: Props) => {
...
-                  margin={EnumFlexItemMargin.Top}
-                  end={<BuildGitLink build={updatedBuild.build} />}
+                  direction={EnumFlexDirection.Column}
+                  gap={EnumGapSize.None}
                 >
-                  <Text textStyle={EnumTextStyle.Tag}>
-                    {updatedBuild.build.message}
-                  </Text>
+                  <Link
+                    target="_blank"
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/VersionControl/BuildPage.tsx:135]:
```suggestion
```

**Review by yuval-hazaz — COMMENTED** [review]
I see that in the design, the request is to open the links in a new tab, but the experience is not fine because the entire page loads when opening in a new tab - so I remove the _blank attribute and will approve

**Review by yuval-hazaz — APPROVED** [review]

**Comment by yuval-hazaz** [comment]
Thanks @sanskar-soni-9 
good job!

**Pull request merged**